.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
how nice! i should try out direnv, shouldn't i? :slightly_smiling_face:

Reply:
The `.envrc` file should instead be `.envrc.example`: otherwise, anyone with direnv installed will be forced to run `use flake` even if they don't use nix.